src/test_report.py

```diff
@@ -26,7 +29,37 @@ def __init__(self, configs, args):
         self._db_config = configs['db_configs'][args.db_config]
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
-        self._logger = Logger("config/logger.conf", "test_report")
+        self._email_hostname = args.email_hostname
+        self._email_hostport = args.email_hostport
+        self._email_password = args.email_password
+        self._email_user = args.email_user
+
+    def create_email(self, template):
+        if template:
+            email_msg = email.mime.text.MIMEText(template, "plain", "utf-8")
+            email_msg.replace_header('content-transfer-encoding', 'quopri')
+            email_msg.set_payload(template, 'utf-8')
+            email_msg['To'] = os.getenv('EMAIL_TO')
+            email_msg['From'] = os.getenv('EMAIL_FROM')
+            email_msg['Subject'] = os.getenv('EMAIL_SUBJECT')
```

Comment:
All variables needed to actually compose the e-mail should be passed to the method as parameters rather than being read from the environment variables inside the method's body. This gives us more flexibility in case we'd like to e.g. use config file or a database instead of env variables.

I'd assume method's signature should be 

```python
def create_email(self, content, to, from, subject):
```